status/discover.go

```diff
@@ -60,18 +80,21 @@ func discoverUrlService(ns string, service string) (url string, err error) {
 	}
 	// If the client is not openshift return and avoid discover
 	if !client.IsOpenShift() {
+		log.Debugf("Kiali detect a no Openshift Cluster")
```

Comment:
I don't think we need all of these debug messages in here - perhaps make them Trace level if you want to keep them. This one doesn't explain where in the processing it is - so seeing this message "not in OpenShift" will be confusing because its not obvious why this debug message is saying this. Also, reword this message... say something like "Discovering external service within a non-OpenShift cluster". That way at least people reading the log will know this message is coming from an attempt to auto-discover jaeger or grafana or some other external service.